CMakeLists.txt

```diff
@@ -39,6 +39,15 @@ else()
 endif()
 # ------------------
 
+# ------------------
+# Extensive mode for running extension one API tests
+option(SYCL_CTS_ENABLE_EXTENSION_ONE_API_TESTS
+       "Enable extension one API tests" OFF)
+if(SYCL_CTS_ENABLE_EXTENSION_ONE_API_TESTS)
+    message(WARNING "Enable extension tests mode: ON")
```

Comment:
```suggestion
    message(STATUS "oneAPI extension tests mode: ON")
```

Reply:
this comment was applied in this commit: ca0b61742388852edfb295a4733924e19be9733c